evalml/pipelines/components/transformers/preprocessing/featuretools.py

```diff
@@ -42,6 +45,24 @@ def _make_entity_set(self, X):
             es = ft_es.add_dataframe(dataframe=X, dataframe_name="X", index=self.index)
         return es
 
+    def _filter_features(self, X):
+        features_to_use = []
+        for feature in self.features:
+            input_cols = [f.column_name for f in feature.base_features]
+            if (
```

Comment:
Might be helpful to add a brief comments about what exactly you are checking in each of the `if` conditions so people revisiting this in the future will be able to follow the logic a little easier.

For example: `# If input columns are not present in X, skip feature`